csharp-basic-topics/ValidateNumberIsPositiveOrNegative/ValidateNumberIsPositiveOrNegative/NumberValidation.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ValidateNumberIsPositiveOrNegative
+{
+    public static class NumberValidation
+    {
+        public static int IsPositiveOrNegativeUsingConditionalMethod(int number)
+        {
+            var result = 0;
+
+            if (number > 0)
+            {
+                result = 1;
+            }
+            else if (number < 0)
+            {
+                result = - 1;
+            }
+            
+            return result;
+        }
+
+        public static int IsPositiveOrNegativeUsingLeftShiftMethod(int number)
+        {
+            var result = 0;
+
+            // Using left shift operator to check the sign bit
```

Comment:
Put this comment before 2nd if statement where we are actually using shift operator. Same comment for right shift

Reply:
Updated comment.